assets/app/view/create_game.rb

```diff
@@ -494,33 +465,37 @@ def filtered_rule?(rule)
     end
 
     def update_inputs(title_change: false)
+      return unless selected_game_or_variant
+
       title = selected_game_or_variant.title
       max_p = @max_p[title]
       min_p = @min_p[title]
-      max_players_elm = Native(@inputs[:max_players]).elm
-      min_players_elm = Native(@inputs[:min_players]).elm
+      if selected_game_or_variant
```

Comment:
Returns on line 468 if selected_game_or_variant is false, so I don't think this check is needed.